cookbook/live-preview.rst

```diff
@@ -0,0 +1,171 @@
+How to include live-preview in my template?
+===========================================
+
+Sulu provides a powerfull live-preview system which can be used by every website
+theme. But as a prerequisite the template has to be adapted a little bit.
+
+Quick Example
+-------------
+
+The preview-system relies on the schema definition from
+`RDFA <https://en.wikipedia.org/wiki/RDFa>`_. This standard allows you to add
+information about the content in the html structure without maintaining another
+file.
+
+The following example is a very basic template implementation which is able to
+be updated by sulu.
+
+**template.html.twig**
+
+.. code-block:: twig
+
+    {% extends "::master.html.twig" %}
+
+    {% block content %}
+        <div id="content" vocab="http://schema.org/" typeof="Content">
+            <h1 property="title">{{ content.title }}</h1>
+        </div>
+    {% endblock %}
+
+**master.html.twig**
+
+.. code-block:: twig
+
+    <!DOCTYPE html>
+    <html>
+        <head>
+            <title>{{ content.title }}
+        </head>
+        <body>
+            {% block content %}{% endblock %}
+        </body>
+    </html>
+
+**page.xml**
+
+.. code-block:: xml
+
+    <?xml version="1.0" ?>
+    <template xmlns="http://schemas.sulu.io/template/template"
+              xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:schemaLocation="http://schemas.sulu.io/template/template http://schemas.sulu.io/template/template-1.0.xsd">
+
+        <key>page</key>
+
+        <view>ClientWebsiteBundle:templates:default</view>
+        <controller>SuluWebsiteBundle:Default:index</controller>
+        <cacheLifetime>2400</cacheLifetime>
+
+        <properties>
+            <section name="highlight">
+                <properties>
+                    <property name="title" type="text_line" mandatory="true">
+                        <params>
+                            <param name="headline" value="true"/>
+                        </params>
+
+                        <tag name="sulu.rlp.part"/>
+                    </property>
+
+                    <property name="url" type="resource_locator" mandatory="true">
+                        <tag name="sulu.rlp"/>
+                    </property>
+                </properties>
+            </section>
+        </properties>
+    </template>
+
+This page consists of a single title and a url. The template is quite basic but
+includes the RDFa attribute `property=title` to tell sulu that the title is
+rendered in this container.
+
+When the title of the page will be updated in the content form the admin will
+send this change to the server where the twig block `content` will be rendered.
+The server response only the part which was changed (indicated by
```

Comment:
I would write `The server returns only the part ...` or `The server response contains only the part ...` but it sounds quite confusing right now.